utils/property-structs.stanza

```diff
@@ -136,4 +136,16 @@ public pcb-struct ocdb/property-structs/CrystalResonator :
 ;     property(obj.vo-diff) = vo-diff 
 ;     property(obj.vo-cm) = vo-cm   
 ;     property(obj.vi-diff) = vi-diff 
-;     property(obj.vi-cm) = vi-cm   
\ No newline at end of file
+;     property(obj.vi-cm) = vi-cm   
+
+; === Property Struct Printers ===
+; ================================
+;[TODO] Write print methods for other property structs.
+defmethod print (o:OutputStream, i:Interval) :
+  print{o, _} $
+    match(nominal(i)) :
+      (nominal:Double) : "[%_, (%_), %_]" % [min(i), nominal, max(i)]
+      (f:False) : "[%_, %_]" % [min(i), max(i)]
+
+defmethod print (o:OutputStream, r:RatedTemperature) :
+  print(o, operating-temperature(r))
```

Comment:
Alternate printers are welcome!
```
Interval(1.0, 2.0)
> [1.0, 2.0]

Interval(1.0, 1.5, 2.0)
> [1.0, (1.5), 2.0]

RatedTemperature(Interval(5.0, 7.0))
> [5.0, 7.0]
```

Reply:
There is a printer in tolerance.stanza for Toleranced() objects. Not sure that we need it here?

Reply:
Right, we won't need another printer for `Toleranced`.
We should still have one for `RatedTemperature`.

Reply:
Oh, I see. RatedTemperature is a type Toleranced now.

Reply:
So all we need now is:
```
defmethod print (o:OutputStream, r:RatedTemperature) : 
  print(o, operating-temperature(r))
```

Reply:
Added!